assets/scss/components/_avatar.scss

```diff
@@ -0,0 +1,170 @@
+}
+
+%avatarCharm {
+	overflow: visible;
+
+	&:after {
+		background-position: center;
+		background-repeat: no-repeat;
+		background-size: 10px;
+		border-radius: 999px;
+		bottom: 0;
+		content: "";
+		display: inline-block;
+		font-size: 10px;
+		font-weight: $W_bold;
+		height: $space;
+		line-height: $space;
+		position: absolute;
+		right: 0;
+		text-align: center;
+		text-indent: 0%;
+		width: $space;
+	}
+}
+
+%avatar,
+.avatar {
```

Comment:
Why are both `%avatar` and `.avatar` needed here? [Looking at this guide](http://thesassway.com/intermediate/understanding-placeholder-selectors) makes me believe that only `.avatar` would be needed.